components/molecules/MarkdownWrapper/markdown-wrapper.tsx

```diff
@@ -0,0 +1,39 @@
+import ReactMarkdown from "react-markdown";
+import SyntaxHighlighter from "react-syntax-highlighter";
+import { atomOneDark } from "react-syntax-highlighter/dist/cjs/styles/hljs";
+
+interface MarkdownWrapperProps {
+  content: string;
+  pClassName?: string;
+}
+
+const MarkdownWrapper = ({ content, pClassName }: MarkdownWrapperProps) => (
+  <ReactMarkdown
+    // eslint-disable-next-line react/no-children-prop
+    children={content}
```

Comment:
do you get any error if you give the `ReactMarkdown` a closing tag ?

```javascript
 <ReactMarkdown
   // {...restProps}
  >
    {children}
 </ReactMarkdown>  
```

Reply:
Yes it can be nested, I just personally found that passing children as props enhances readability in this case. But it does make sense to nest in order to remove the ESlint warnings. That change has been committed, thank you.